### PR TITLE
More and more changes in ONE single pr

### PR DESCRIPTION
**Pull request by karmaisblackandbluepilled**
FIXES:
Ports [#65783](https://github.com/tgstation/tgstation/pull/65783) from tg because it's a fix
Ports [#65781](https://github.com/tgstation/tgstation/pull/65781) for the same reason
Ports [#65744](https://github.com/tgstation/tgstation/pull/65744) for the same reason
REVERTS:
Re-adds self-surgery. Higher chance of failure and always a 5% chance and slower than normal surgery.
Crayons colour you when you eat them again.
Reverts [#44172](https://github.com/tgstation/tgstation/pull/44172)
Reverts [#50721](https://github.com/tgstation/tgstation/pull/50721)
Reverts [47084](https://github.com/tgstation/tgstation/pull/47084) why would you nerf the wishgranter?
Flashes stun you again, no more stamloss shit.
Adds vitiligo and revitiligo, removes disfigured and polyvitiligo.
Reverts 25422, from 2017.
Removes misfire increment added in [56322](https://github.com/tgstation/tgstation/pull/56322)
Reverts the "drop items" AND SPEED NERF FORGOT TO SAY THIS HERE BEFORE part of [42553 ](https://github.com/tgstation/tgstation/pull/42553)
Reverts quint/quadsec nerfs
OH HI DANIEL IS BACK BABYYYYY
Adds loot to syndicate mobs again.
Buffs spray cleaner back to previous volume.
Reverts [45274](https://github.com/tgstation/tgstation/pull/45274)
Reverts [41151](https://github.com/tgstation/tgstation/pull/41151)
Medbots should heal every damage type again now.
Reverts unsentienting mobs with slimes why?
Yellow slime cores should work as before now.
ADDITIONS(that aren't from reverting):
Added more creatures to the golden slime(who the fuck still does xenobio)

### CODE REVIEW

**Comment by trollertroll** [comment]
devil: my son will properly document his changes
jesus:

**Comment by karmaisblackandbluepilled** [comment]
Monster PR ready for review :hownice:

**Comment by EdgeLordExe** [comment]
The only thing i dislike is the flash thing as it was basicallly an old stun baton, bring back stun bur keep the directional mechanics

**Comment by AcapitalA** [comment]
TCEO is already working with sec equipment, including flashes.
Also, Oh Hi Daniel was already in our codebase if you emagged the command console.

**Comment by trollertroll** [comment]
don't care + didn't ask + cry about it + stay mad + get real + L + mald seethe cope harder + hoes mad + basic + skill issue + ratio + you fell off + the audacity + triggered + any askers + bluepilled + get a life + ok and? + cringe + touch grass + donowalled + not based + you're a basic bitch + not funny didn't laugh + you're* + grammar issue + go outside + get good + reported + ad hominem + motherless + fatherless + adopted + sugma + ligma + failed abortion + die + pokimane simp + uncleless + Your dad isnt coming back + homo + furry lover + GG !

**Comment by karmaisblackandbluepilled** [comment]
> The only thing i dislike is the flash thing as it was basicallly an old stun baton, bring back stun bur keep the directional mechanics

stun bur?

**Comment by trollertroll** [comment]
> > The only thing i dislike is the flash thing as it was basicallly an old stun baton, bring back stun bur keep the directional mechanics
> 
> stun bur?

i think he meant to say butt

**Comment by karmaisblackandbluepilled** [comment]
> TCEO is already working with sec equipment, including flashes. Also, Oh Hi Daniel was already in our codebase if you emagged the command console.

yeah but I don't think it should need an emag, I think it should be free because it's so fucking ~~bad~~ good

File: code/__DEFINES/mobs.dm [hunk 1]
```diff
@@ -224,7 +224,7 @@
 //Sentience types, to prevent things like sentience potions from giving bosses sentience
 #define SENTIENCE_ORGANIC 1
 #define SENTIENCE_ARTIFICIAL 2
-#define SENTIENCE_HUMANOID 3
+// #define SENTIENCE_OTHER 3 unused
```
Comment (TheChosenEvilOne) [inline code/__DEFINES/mobs.dm:227]: if this isn't used, just remove it. also change the numbers below

File: code/datums/diseases/advance/symptoms/disfiguration.dm [hunk 2]
```diff
@@ -1,50 +0,0 @@
-/*
```
Comment (TheChosenEvilOne) [inline code/datums/diseases/advance/symptoms/disfiguration.dm:1]: why remove this symptom?

File: code/datums/shuttles.dm [hunk 3]
```diff
@@ -407,7 +407,6 @@
 	admin_notes = "Tiny, single airlock, wooden walls. No way anyone would maxcap the thing..."
 	movement_force = list("KNOCKDOWN" = 3, "THROW" = 2)
 	credit_cost = CARGO_CRATE_VALUE * 4
-	restricted = TRUE
```
Comment (TheChosenEvilOne) [inline code/datums/shuttles.dm:410]: fine, but this part is getting reverted if it is done too many times.

File: code/game/machinery/computer/arcade/orion.dm [hunk 4]
```diff
@@ -492,8 +492,26 @@ GLOBAL_LIST_INIT(orion_events, generate_orion_events())
...
+	name = "Orion Spaceport Security"
+	uniform = /obj/item/clothing/under/syndicate
+	suit = /obj/item/clothing/suit/armor/vest
+	shoes = /obj/item/clothing/shoes/combat
+	gloves = /obj/item/clothing/gloves/combat
+	ears = /obj/item/radio/headset
+	mask = /obj/item/clothing/mask/gas
+	head = /obj/item/clothing/head/helmet/swat
+	back = /obj/item/storage/backpack
+	id = /obj/item/card/id/
```
Comment (TheChosenEvilOne) [inline code/game/machinery/computer/arcade/orion.dm:513]: should probably be a chameleon card

File: code/game/objects/items/storage/firstaid.dm [hunk 5]
```diff
@@ -256,7 +256,6 @@
 	icon_state = "radfirstaid"
 	inhand_icon_state = "firstaid-rad"
 	custom_premium_price = PAYCHECK_HARD * 6
-	damagetype_healed = "all"
```
Comment (TheChosenEvilOne) [inline code/game/objects/items/storage/firstaid.dm:259]: remove the damagetype_healed var

File: code/modules/mob/living/simple_animal/bot/medbot.dm [hunk 6]
```diff
@@ -51,8 +51,8 @@
 	var/heal_amount = 2.5
 	/// Start healing when they have this much damage in a category
 	var/heal_threshold = 10
-	/// What damage type does this bot support. Because the default is brute, if the medkit is brute-oriented there is a slight bonus to healing. set to "all" for it to heal any of the 4 base damage types
-	var/damagetype_healer = BRUTE
+	/// What damage type does this bot support. FUCK FUCK FUCK YOU
+	var/damagetype_healer = ALL
```
Comment (TheChosenEvilOne) [inline code/modules/mob/living/simple_animal/bot/medbot.dm:55]: remove this completely.

File: code/modules/mob/living/simple_animal/hostile/bosses/paperwizard.dm [hunk 7]
```diff
@@ -2,7 +2,7 @@
 /mob/living/simple_animal/hostile/boss/paper_wizard
 	name = "Mjor the Creative"
 	desc = "A wizard with a taste for the arts."
-	mob_biotypes = MOB_ORGANIC|MOB_HUMANOID
+	mob_biotypes = MOB_HUMANOID
```
Comment (TheChosenEvilOne) [inline code/modules/mob/living/simple_animal/hostile/bosses/paperwizard.dm:5]: hm?

File: code/modules/mob/living/simple_animal/hostile/dark_wizard.dm [hunk 8]
```diff
@@ -22,8 +22,7 @@
 	attack_sound = 'sound/weapons/bladeslice.ogg'
 	aggro_vision_range = 9
 	turns_per_move = 5
-	mob_biotypes = MOB_ORGANIC|MOB_HUMANOID
```
Comment (TheChosenEvilOne) [inline code/modules/mob/living/simple_animal/hostile/dark_wizard.dm:25]: huh?

File: code/modules/power/cell.dm [hunk 9]
```diff
@@ -83,6 +83,25 @@
...
+			else
+				STOP_PROCESSING(SSobj, src)
+	. = ..()
+
+/obj/item/stock_parts/cell/process(delta_time)
+	if(self_recharge)
+		give(chargerate * 0.125 * delta_time)
+	else
+		return PROCESS_KILL
+
```
Comment (TheChosenEvilOne) [inline code/modules/power/cell.dm:104]: this has already been reverted.

File: code/modules/power/cell.dm [hunk 10]
```diff
@@ -393,6 +412,18 @@
...
+	desc = "A yellow slime core infused with plasma, it crackles with power."
+	icon = 'icons/mob/slimes.dmi'
+	icon_state = "yellow slime extract"
+	custom_materials = null
+	rating = 5 //self-recharge makes these desirable
+	self_recharge = 1 // Infused slime cores self-recharge, over time
+
+/*Hypercharged slime cell - located in /code/modules/research/xenobiology/crossbreeding/_misc.dm
+/obj/item/stock_parts/cell/high/slime/hypercharged */
+
```
Comment (TheChosenEvilOne) [inline code/modules/power/cell.dm:426]: ditto.

File: code/modules/research/designs/medical_designs.dm [hunk 11]
```diff
@@ -110,6 +120,16 @@
...
 
+/datum/design/noreactsyringe
+	name = "Cryo Syringe"
+	desc = "An advanced syringe that stops reagents inside from reacting. It can hold up to 20 units."
+	id = "noreactsyringe"
+	build_type = PROTOLATHE
+	materials = list(/datum/material/glass = 2000, /datum/material/gold = 1000)
+	build_path = /obj/item/reagent_containers/syringe/noreact
+	category = list("Medical Designs")
+	departmental_flags = DEPARTMENTAL_FLAG_MEDICAL
```
Comment (TheChosenEvilOne) [inline code/modules/research/designs/medical_designs.dm:131]: why is this added twice.

File: code/modules/research/designs/weapon_designs.dm [hunk 12]
```diff
@@ -322,6 +332,16 @@
...
 
+/datum/design/cryostasis_shotgun_dart
+	name = "Cryostasis Shotgun Dart"
+	desc = "A shotgun dart designed with similar internals to that of a cryostasis beaker, allowing reagents to not react when inside."
+	id = "shotgundartcryostasis"
+	build_type = PROTOLATHE
+	materials = list(/datum/material/iron = 3500)
+	build_path = /obj/item/ammo_casing/shotgun/dart/noreact
+	category = list("Ammo")
+	departmental_flags = DEPARTMENTAL_FLAG_MEDICAL
```
Comment (TheChosenEvilOne) [inline code/modules/research/designs/weapon_designs.dm:343]: ditto

File: code/modules/ruins/spaceruin_code/forgottenship.dm [hunk 13]
```diff
@@ -172,6 +172,7 @@ GLOBAL_VAR_INIT(fscpassword, generate_password())
 /mob/living/simple_animal/hostile/nanotrasen/elite
 	name = "Nanotrasen Elite Assault Officer"
 	desc = "Pray for your life, syndicate. Run while you can."
+	icon = 'icons/mob/simple_human.dmi'
```
Comment (TheChosenEvilOne) [inline code/modules/ruins/spaceruin_code/forgottenship.dm:175]:
```suggestion
```

File: code/modules/surgery/organ_manipulation.dm [hunk 14]
```diff
@@ -39,7 +39,6 @@
 	possible_locs = list(BODY_ZONE_CHEST, BODY_ZONE_HEAD)
 	requires_bodypart_type = BODYPART_ROBOTIC
 	lying_required = FALSE
-	self_operable = TRUE
```
Comment (TheChosenEvilOne) [inline code/modules/surgery/organ_manipulation.dm:42]: I am going to OK this ONLY if you disallow self-surgery on head and arms.

**Review by TheChosenEvilOne — CHANGES_REQUESTED** [review]
h

File: code/datums/diseases/advance/symptoms/disfiguration.dm (hunk 2 above)
Comment (karmaisblackandbluepilled) [reply code/datums/diseases/advance/symptoms/disfiguration.dm:1]: mainly because it was added as a replacement to vitiligo/revitiligo, so I just removed it with the rest when I added the two back in

File: code/game/area/space_station_13_areas.dm [hunk 15]
```diff
@@ -98,16 +98,9 @@ NOTE: there are two lists of areas in the end of this file: centcom and station
 //AI - Turret_protected
 
 /area/ai_monitored/turret_protected
-	ambientsounds = list('sound/ambience/ambitech.ogg', 'sound/ambience/ambitech2.ogg', 'sound/ambience/ambiatmos.ogg', 'sound/ambience/ambiatmos2.ogg')
-	///Some sounds (like the space jam) are terrible when on loop. We use this varaible to add it to other AI areas, but override it to keep it from the AI's core.
-	var/ai_will_not_hear_this = list('sound/ambience/ambimalf.ogg')
+	ambientsounds = list('sound/ambience/ambimalf.ogg', 'sound/ambience/ambitech.ogg', 'sound/ambience/ambitech2.ogg', 'sound/ambience/ambiatmos.ogg', 'sound/ambience/ambiatmos2.ogg')
```
[the remark this answers is not in the document]
Comment (karmaisblackandbluepilled) [reply code/game/area/space_station_13_areas.dm:101]: it's the uhhhhhh "EVERYBODY GET UP, IT'S TIME TO SLAM NOW" ambient sound from the AI sat that was removed for some retarded reason

File: code/game/machinery/computer/arcade/orion.dm (hunk 4 above)
Comment (karmaisblackandbluepilled) [reply code/game/machinery/computer/arcade/orion.dm:513]: ok

File: code/modules/research/designs/medical_designs.dm (hunk 11 above)
Comment (karmaisblackandbluepilled) [reply code/modules/research/designs/medical_designs.dm:131]: because I suck ass at reverting

File: code/modules/mob/living/simple_animal/hostile/dark_wizard.dm (hunk 8 above)
Comment (karmaisblackandbluepilled) [reply code/modules/mob/living/simple_animal/hostile/dark_wizard.dm:25]: revert fuckup(nr 200)

File: code/modules/reagents/reagent_containers/spray.dm [hunk 16]
```diff
@@ -211,7 +211,6 @@
 	name = "space cleaner"
 	desc = "BLAM!-brand non-foaming space cleaner!"
 	icon_state = "cleaner"
-	volume = 100
```
[the remark this answers is not in the document]
Comment (karmaisblackandbluepilled) [reply code/modules/reagents/reagent_containers/spray.dm:214]: yeah uhhhh 200 instead of 250 then :clueless:

File: code/game/area/space_station_13_areas.dm (hunk 15 above)
[the remark this answers is not in the document]
Comment (karmaisblackandbluepilled) [reply code/game/area/space_station_13_areas.dm:101]: wait did I fucking revert the wrong pr

**Review by karmaisblackandbluepilled — COMMENTED** [review]
oh god what the fuck is this I thought I was sending a message help

**Comment by karmaisblackandbluepilled** [comment]
ok no on further checks that's a correct pr revert, the ambimalf shit was removed from inside the ai core room, bad! I fix!

File: code/game/area/space_station_13_areas.dm (hunk 15 above)
[the remark this answers is not in the document]
Comment (karmaisblackandbluepilled) [reply code/game/area/space_station_13_areas.dm:101]: nah we good

File: code/modules/surgery/organ_manipulation.dm (hunk 14 above)
Comment (karmaisblackandbluepilled) [reply code/modules/surgery/organ_manipulation.dm:42]: Nah, but I changed it to take 3 times as long and always have a min failing chance of 5%. No reason to block head and arms. Merge merge merge

File: code/datums/diseases/advance/symptoms/disfiguration.dm (hunk 2 above)
Comment (TheChosenEvilOne) [reply code/datums/diseases/advance/symptoms/disfiguration.dm:1]: this is a more interesting symptom IMO, so I'd like to keep it as well. though I think I will have to ask some viro players for opinions about this.

File: code/modules/surgery/surgery_step.dm [hunk 17]
```diff
@@ -12,6 +12,8 @@
 
 /datum/surgery_step/proc/try_op(mob/user, mob/living/target, target_zone, obj/item/tool, datum/surgery/surgery, try_to_fail = FALSE)
 	var/success = FALSE
+	if(user == target)
+		time = time*3
```
Comment (TheChosenEvilOne) [inline code/modules/surgery/surgery_step.dm:16]:
```suggestion
		time *= 3
```

File: code/modules/surgery/organ_manipulation.dm (hunk 14 above)
Comment (TheChosenEvilOne) [reply code/modules/surgery/organ_manipulation.dm:42]: > Nah, but I changed it to take 3 times as long and always have a min failing chance of 5%. No reason to block head and arms. Merge merge merge

I still want people to be dependent on other people to do surgery in some cases, how about head being disallowed when doing self surgery?

File: code/modules/surgery/surgery_step.dm [hunk 18]
```diff
@@ -77,12 +79,13 @@
...
-
 	speed_mod /= (get_location_modifier(target) * (1 + surgery.speed_modifier) * implement_speed_mod) * target.mob_surgery_speed_mod
 	var/modded_time = time * speed_mod
 
-
-	fail_prob = min(max(0, modded_time - (time * SURGERY_SLOWDOWN_CAP_MULTIPLIER)),99)//if modded_time > time * modifier, then fail_prob = modded_time - time*modifier. starts at 0, caps at 99
+	if(user == target)
+		fail_prob = min(max(0, modded_time - time),99)
+	else
+		fail_prob = max(5, min(max(0, modded_time - (time * SURGERY_SLOWDOWN_CAP_MULTIPLIER)),99))//if modded_time > time * modifier, then fail_prob = modded_time - time*modifier. starts at 0, caps at 99
```
Comment (TheChosenEvilOne) [inline code/modules/surgery/surgery_step.dm:88]: buhh?? why does doing surgery to other people have a 5% chance to fail?

Comment (karmaisblackandbluepilled) [reply code/modules/surgery/surgery_step.dm:88]: because I'm a fucking retard

File: code/modules/surgery/surgery_step.dm (hunk 17 above)
Comment (karmaisblackandbluepilled) [reply code/modules/surgery/surgery_step.dm:16]: oh I didn't know you could write it like that

File: code/datums/diseases/advance/symptoms/disfiguration.dm (hunk 2 above)
Comment (karmaisblackandbluepilled) [reply code/datums/diseases/advance/symptoms/disfiguration.dm:1]: yeah ok I added it back

File: code/modules/surgery/organ_manipulation.dm (hunk 14 above)
Comment (EdgeLordExe) [reply code/modules/surgery/organ_manipulation.dm:42]: self surgery should be gruelingly hard, give it a base 25% chance of failure minimum.

**Review by TheChosenEvilOne — APPROVED** [review]
on the code side seems ok.

**Comment by TheChosenEvilOne** [comment]
webhook test

File: code/modules/surgery/surgery_step.dm [hunk 19]
```diff
@@ -77,12 +81,13 @@
...
 	if(implement_type) //this means it isn't a require hand or any item step.
 		implement_speed_mod = implements[implement_type] / 100.0
-
 	speed_mod /= (get_location_modifier(target) * (1 + surgery.speed_modifier) * implement_speed_mod) * target.mob_surgery_speed_mod
 	var/modded_time = time * speed_mod
 
-
-	fail_prob = min(max(0, modded_time - (time * SURGERY_SLOWDOWN_CAP_MULTIPLIER)),99)//if modded_time > time * modifier, then fail_prob = modded_time - time*modifier. starts at 0, caps at 99
+	if(user == target)
+		fail_prob = max(20, min(max(0, modded_time - (time * SURGERY_SLOWDOWN_CAP_MULTIPLIER)),99))
```
Comment (TheChosenEvilOne) [inline code/modules/surgery/surgery_step.dm:88]:
```suggestion
		fail_prob = min(max(20, modded_time - (time * SURGERY_SLOWDOWN_CAP_MULTIPLIER)),99)
```

**Review by EdgeLordExe — APPROVED** [review]
